olympus/drivers/sounds.c

```diff
@@ -0,0 +1,118 @@
+#include "sounds.h"
+#include "tim.h"
+#include "stm32f2xx_hal.h"
+#include "timerCallback.h"
+#include <stdint.h>
+
+//these are the correct counts for these particular notes
+#define C4 5707  
+#define c4 5386
+#define D4 5084
+#define d4 4798
+#define E4 4529
+#define F4 4275
+#define f4 4035
+#define G4 3808
+#define g4 3595
+#define A4 3393
+#define a4 3202
+#define B4 3023
+#define C5 2853
+#define c5 2693
+#define D5 2542
+#define d5 2399
+#define E5 2265
+#define F5 2138
+#define f5 2017
+#define G5 1904
+#define g5 1797
+#define A5 1696
+#define a5 1601
+#define B5 1511
+#define C6 1426
+#define c6 1346
+#define D6 1271
+#define d6 1199
+#define E6 1132
+#define g7 899
+
+note windows[]= {  //notes and lengths needed to play windows start up sound
+    {d6, 3},
+    {d5, 1},
+    {a5, 4},
+    {g5, 5},
+    {d6, 3},
+    {a5, 8}
+    };
+
+note shutdown[]={   //plays the windows shutdown sound 
+    {g7, 3},
+    {d6, 3},
+    {g5, 3},
+    {a5, 4}
+    };
+  
+uint8_t idx = 0; //sets the index to 0
+
+uint16_t noteslen[13] = {
+    0, 107, 214, 321, 428, 535, 642, 750, 857, 964, 1071, 1178, 1285};
+    //to calculate length use #16th notes * .25 * (1/tempo)*1000*60
+
+Timer_Return startup(void*dummy)//creates a call-back timer that will check back in
+{
+    
+    if (idx < sizeof(windows)/sizeof(note )){
+        playNote(windows[idx].freq); //play the note give by the struct
```

Comment:
Doesn't match indentation of next line. The next two lines should have four spaces